src/wp-includes/SimplePie/Item.php

```diff
@@ -122,7 +122,7 @@ public function __toString()
 	 */
 	public function __destruct()
 	{
-		if ((version_compare(PHP_VERSION, '5.3', '<') || !gc_enabled()) && !ini_get('zend.ze1_compatibility_mode'))
+		if (!ini_get('zend.ze1_compatibility_mode'))
```

Comment:
Let's back out all changes to SimplePie (and any other third-party libraries) for now.

We'll need to investigate these independently:

- Whether WordPress has kept them up to date, or made their own custom changes, or some combination of the above
- How much effort and risk would be involved in updating our copies of these libraries ourselves

See also: https://github.com/ClassicPress/ClassicPress/pull/146#pullrequestreview-168499933